src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -644,6 +644,72 @@ doLedgerEntry(RPC::JsonContext& context)
                     uNodeIndex = keylet::oracle(*account, *documentID).key;
             }
         }
+        else if (context.params.isMember(jss::mpt_issuance))
```

Comment:
Not your fault, but this is the first time I've seen this function, and what in the hell is going on here? 700 line function to manually type check the return value? Is this necessary? Is there no better way?